client/package.json

```diff
@@ -10,53 +10,53 @@
     "lint:nofix": "vue-cli-service lint --no-fix"
   },
   "dependencies": {
-    "axios": "^0.19.0",
+    "@mdi/font": "^4.5.95",
+    "axios": "^0.19.2",
     "babel-plugin-transform-decorators": "^6.24.1",
-    "core-js": "^3.3.2",
+    "core-js": "^3.7.0",
     "lodash": "^4.17.20",
-    "@mdi/font": "^4.5.95",
     "moment": "^2.27.0",
     "remove-accents": "^0.4.2",
     "sbc-common-components": "2.4.5",
     "style-loader": "^1.2.1",
-    "vue": "^2.6.10",
-    "vue-class-component": "^7.0.2",
+    "vue": "^2.6.11",
     "vue-contenteditable": "^1.0.2",
-    "vue-property-decorator": "^8.3.0",
     "vue-quill-editor": "^3.0.6",
-    "vue-router": "^3.1.3",
-    "vuetify": "^2.1.0",
-    "vuex": "^3.0.1",
+    "vue-router": "^3.4.9",
+    "vuetify": "^2.3.18",
+    "vuex": "^3.5.1",
     "vuex-class": "^0.3.2"
   },
   "devDependencies": {
-    "@types/jest": "^25.1.4",
-    "@types/sinon": "^7.5.1",
+    "@types/jest": "^25.2.3",
+    "@types/sinon": "^7.5.2",
     "@vue/cli-plugin-babel": "^4.4.1",
-    "@vue/cli-plugin-eslint": "^4.0.0",
-    "@vue/cli-plugin-router": "^4.0.0",
-    "@vue/cli-plugin-typescript": "^4.3.1",
-    "@vue/cli-plugin-unit-jest": "^4.2.3",
-    "@vue/cli-plugin-vuex": "^4.0.0",
-    "@vue/cli-service": "^4.2.3",
+    "@vue/cli-plugin-eslint": "^4.4.1",
+    "@vue/cli-plugin-router": "^4.4.1",
+    "@vue/cli-plugin-typescript": "^4.4.1",
+    "@vue/cli-plugin-unit-jest": "^4.4.1",
+    "@vue/cli-plugin-vuex": "^4.4.1",
+    "@vue/cli-service": "^4.4.1",
     "@vue/eslint-config-standard": "^4.0.0",
     "@vue/eslint-config-typescript": "^4.0.0",
-    "@vue/test-utils": "^1.0.0-beta.30",
-    "cross-env": "^7.0.0",
+    "@vue/test-utils": "^1.1.1",
```

Comment:
... Except for this one, which is a "significant" jump.

In this project, all the unit tests worked fine with the new version. (I was hoping it would resolve some warnings but it didn't. It appears those warnings are from the component is built or mounted or .../)

In other projects, the unit tests are sensitive to changes between beta versions so this package can't simply (sic) be upgrade simply (sic).